### ISSUE TITLE
mdb_env() locks if another process is running a write transaction

### ISSUE THREAD

**Issue by gaborcsardi** [richfitz/thor#17]
Maybe I misunderstand something, but I think the other process should still be able to do read transactions, no?

**Comment by richfitz** [comment]
Yes, I'd have thought so too. I will look into this when I get a chance
